### PR TITLE
Add data base

### PR DESCRIPTION
**Pull request by Yasmarila**
- I added a link between the data base and the web application
- I added a view for the payment

### CODE REVIEW

File: Web/CSS/pay_article.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    border: 0;
+    vertical-align: middle;
+    text-decoration: none;
+    background: transparent;
+    padding: 0;
+    font-size: inherit;
+    font-family: inherit;
+    width: 12rem;
+    height: auto;
+  }
```
Comment (galyfray) [inline Web/CSS/pay_article.css:130]: indent fail

File: Web/HTML/display_articles.html [hunk 2]
```diff
@@ -4,219 +4,48 @@
...
-                                    <li>
-                                        <a href="#">Réparation carrossrie</a>
-                                    </li>
-                                </ul>
-                            </li>
+                        <ul id="main-nav2">
                         </ul>
                     </nav>
-                    <a class="menu_icon">
+                    <a class="menu_icon" style="">
```
Comment (galyfray) [inline Web/HTML/display_articles.html]: why an empty style

File: Web/HTML/display_articles.html [hunk 3]
```diff
@@ -4,219 +4,48 @@
...
-                <h1>Tout les pneus</h1>
+                <h1 id="all_the_category">Tout les articles</h1>
                 <div id="div_container_all_article"></div>
             </div>
         </div>
     </body>
-</html>
\ No newline at end of file
+</html>
+<script type="module">
```
Comment (galyfray) [inline Web/HTML/display_articles.html]: no JS directly in the HTML + what is this code for ?

File: Web/HTML/pay_article.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+<script src="https://smtpjs.com/v3/smtp.js">
+</script>
+<input type="button" value="Send Email" onclick="sendEmail()">
+<script type="text/javascript">
+    function sendEmail() {
+      Email.send({
+        Host: "smtp.gmail.com",
+        Username: "sender@email_address.com",
+        Password: "Enter your password",
+        To: 'yasmarila.n@gmail.com',
```
Comment (galyfray) [inline Web/HTML/pay_article.html]: mmmmmm

File: Web/JS/create_menu copy.js [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+                //             Console.log("sous catégorie");
+                let subCategory = document.createElement("li");
+                subCategory.id = "li" + val.value[i].header_category_id;
+                subCategory.id = "li2" + val.value[i].header_category_id;
+                subCategory.appendChild(element_content);
+                future_menu.appendChild(subCategory);
+            }
+        }
+
+        //Vide
```
Comment (galyfray) [inline Web/JS/create_menu copy.js]: //Vide

File: Web/JS/display_article_js.js [hunk 6]
```diff
@@ -1,66 +1,188 @@
...
-            customToggle    : '.menu_icon',
-            levelSpacing    : 40,
-            navTitle        : 'All',
-            levelTitles     : true,
-            levelTitleAsBack: true,
-            pushContent     : '#container',
-            labelClose      : false
-        });
-    })(jQuery);
+    //SI VIDE ALORS AFFICHE 0 MAIS SI MEMOIRE AFFICHE VALEUR
```
Comment (galyfray) [inline Web/JS/display_article_js.js]: I DO AGREE

**Review by galyfray — CHANGES_REQUESTED** [review]

File: Web/JS/create_menu copy.js (hunk 5 above)
Comment (Yasmarila) [reply Web/JS/create_menu copy.js]: I don't remember what I put it. I deleted it

File: Web/HTML/pay_article.html (hunk 4 above)
Comment (Yasmarila) [reply Web/HTML/pay_article.html]: I removed my email

File: Web/HTML/display_articles.html (hunk 2 above)
Comment (Yasmarila) [reply Web/HTML/display_articles.html]: Deleted.

File: Web/HTML/display_articles.html (hunk 3 above)
Comment (Yasmarila) [reply Web/HTML/display_articles.html]: Not used anymore

**Review by galyfray — APPROVED** [review]
gonna say Ok

**Pull request merged**